lms/views/exceptions.py

```diff
@@ -50,7 +50,10 @@ def forbidden(self):
                 )
                 return self.serializable_error()
 
-        return self.error_response(403, _("You're not authorized to view this page"))
+        messages = ["You're not authorized to view this page"]
+        if details := getattr(self.exception, "message"):
```

Comment:
What's the purpose of exposing the exception message to end users? Could we end up exposing too much information if an unknown exception is raised somewhere and it ends up being "captured" here?

Reply:
Good point added more notes on the description and changed the approach.